apps/openmw/mwmechanics/aiwander.cpp

```diff
@@ -200,7 +200,18 @@ namespace MWMechanics
             stopWalking(actor, storage);
             currentCell = actor.getCell();
             storage.mPopulateAvailableNodes = true;
-            mStoredInitialActorPosition = false;
```

Comment:
We do not store an AiStorage in a savegame, and I can determine an old cell only by mInitialActorPosition after save load.
Actually, I have never seen cellChange == true during my testing (a batch of console commands from [this](https://bugs.openmw.org/issues/4010) topic):
First execute(): cellChange is false, because there is no currentCell in the AiState yet.
Second execute(): cellChange is false, because actor.getCell() == currentCell already.
Any ideas how to check if a current cell was REALLY changed after last update?